src/core/georeferencing.h

```diff
@@ -137,39 +137,57 @@ friend QDebug operator<<(QDebug dbg, const Georeferencing& georef);
 	
 	
 	/**
-	 * @brief Returns the precision of the grid scale factor.
+	 * @brief Returns the precision of the combined/auxiliary scale factors.
```

Comment:
The comment updates seem unrelated to the number precision topic. If that is the case, I'd like to see them in a separate patch.

Reply:
You're correct. I will rebase and split off this change.

Reply:
Split off 20afafc.